### PR TITLE
Чирков Р.Д. 381806-2. ЛР3

### PR DESCRIPTION
**Pull request by ChirkovRoman**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [ ] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [ ] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [ ] Get two approve from your fellow students
- [ ] Take a review from one of the administrators

### CODE REVIEW

**Comment by dkurt** [comment]
Please rebase with main branch to resolve CI issue

**Comment by ChirkovRoman** [comment]
@netweel @palisarbaro проверьте, пожалуйста

File: modules/complex-calc/src/application.cpp [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  }
+  return op;
+}
+
+Complex parseOperand(const char* argReal, const char* argIm) {
+  double real, im;
+  try {
+    real = stod(std::string(argReal));
+    im = stod(std::string(argIm));
+  } catch (std::invalid_argument& e) {
```
Comment (palisarbaro) [inline modules/complex-calc/src/application.cpp:33]: тут еще может быть out_of_range

File: modules/complex-calc/src/application.cpp [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    throw std::string("Wrong operand.");
+  }
+
+  return Complex(real, im);
+}
+
+Application::Application() {
+  leftOperand = Complex();
```
[the remark this answers is not in the document]
Comment (ChirkovRoman) [reply modules/complex-calc/src/application.cpp:41]: Точно, забыл. Пережитки прошлой версии

**Review by palisarbaro — APPROVED** [review]

**Review by netweel — APPROVED** [review]
perfect

**Review by WerWebWer — APPROVED** [review]

**Pull request merged**